zebra-state/src/request.rs

```diff
@@ -239,4 +239,18 @@ pub enum Request {
     /// Code making this request should apply a timeout layer to the service to
     /// handle missing UTXOs.
     AwaitUtxo(transparent::OutPoint),
+
+    /// Finds the intersection of the block hashes provided by a peer, and the local best chain.
+    /// Returns a list of block hashes, starting at the child of the intersection, and extending towards the tip.
+    ///
+    /// Returns
+    ///
+    /// [`Response::BlockHashes(Vec<block::Hash>)`](Response::Hashes) with max of 500 block hashes.
```

Comment:
Broken link, let's try to do renames using search and replace
```suggestion
    /// [`Response::BlockHashes(Vec<block::Hash>)`](Response::BlockHashes) with max of 500 block hashes.
```